olive/common/ort_inference.py

```diff
@@ -285,7 +285,7 @@ def get_full_input_feed(self, input_feed: Dict[str, "NDArray"]) -> Dict[str, "ND
         """Get the full input feed including constant inputs."""
         return {**input_feed, **self.constant_inputs}
 
-    def run(self, input_feed: Dict[str, "NDArray"]) -> Sequence["NDArray"]:
+    def run(self, output_names, input_feed: Dict[str, "NDArray"], run_options=None) -> Sequence["NDArray"]:
```

Comment:
I forgot about this one. Please update this call here to `session_wrapper.run(None, input_feed)` https://github.com/microsoft/Olive/blob/2230ed39c79829ca42d77cee17d7042488db135c/olive/systems/isolated_ort/inference_runner.py#L79

Reply:
Might have to update this test here too https://github.com/microsoft/Olive/blob/2230ed39c79829ca42d77cee17d7042488db135c/test/unit_test/systems/isolated_ort/test_isolated_ort_system.py#L175

Reply:
Fixed, No change to test. Both tests passed.